src/main/resources/application.yaml

```diff
@@ -16,7 +16,7 @@ spring:
 appealSubmissionDocument.contentType: application/pdf
 appealSubmissionDocument.fileExtension: PDF
 appealSubmissionDocument.fileName: "appeal-form"
-appealSubmissionDocument.templateName: ${IA_APPEAL_SUBMISSION_TEMPLATE:TB-IAC-APP-ENG-00003.docx}
+appealSubmissionDocument.templateName: ${IA_APPEAL_SUBMISSION_TEMPLATE:TB-IAC-APP-ENG-00063.docx}
```

Comment:
When in the office, remind me to show you how we maintain this template config through different environments.

Reply:
Ok, I will do. I thought to change default value to the latest one. Let's catch up on Monday.

Reply:
Yeah I think that's good.